flexmeasures_client.py

```diff
@@ -299,28 +299,21 @@ def trigger_schedule(self, *args, **fnc_kwargs):
 
         if schedule_id is None:
             self.log_failed_response(res, url)
-            self.handle_response_errors(message, res, url, self.trigger_schedule, *args, **fnc_kwargs)
+            self.try_solve_authentication_error(message, res, url, self.trigger_schedule, *args, **fnc_kwargs)
             self.set_state("input_boolean.error_schedule_cannot_be_retrieved", state="on")
             return None
 
         self.log(f"Successfully triggered schedule. Schedule id: {schedule_id}")
         self.set_state("input_boolean.error_schedule_cannot_be_retrieved", state="off")
         return schedule_id
 
-    def handle_response_errors(self, message, res, description, fnc, **fnc_kwargs):
+    def try_solve_authentication_error(self, message, res, description, fnc, **fnc_kwargs):
```

Comment:
The function signature should be updated:
- `message` is no longer used
- `description` could be updated to `url` (also updating the log statement: `"Call to {url} failed on authorization ..."`)
-  `*args` are passed; they should be picked up here, I suggest as `*fnc_args` (this already seemed to be a problem before this PR)

In conjunction, the call to this function should be updated by removing the `message` variable.

Reply:
Done